test/logoutrequest_test.rb

```diff
@@ -152,21 +152,7 @@ class RequestTest < Minitest::Test
         assert_match %r[<ds:DigestMethod Algorithm='http://www.w3.org/2001/04/xmlenc#sha256'/>], inflated
       end
 
-      it "create a signed logout request" do
-        settings.compress_request = true
```

Comment:
@johnnyshields 
Should we instead remove only the:
```
settings.compress_request = true
```
and use
```
settings.idp_slo_service_binding = RubySaml::Utils::BINDINGS[:post]
```
So we keep the test?

Reply:
You have very good eyes! 👀

It was removed because when you remove compression from this case (as required for HTTP-POST) and test the `create_params` method (`create` is not used for POST), then it becomes identical to the test immediately below it.